assig2/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import * as serviceWorker from './serviceWorker';
+import 'bootstrap/dist/css/bootstrap.css';
```

Comment:
In "real life" you would try to avoid at all costs copying code from libraries in your git. Instead, you would use npm to install bootstrap and include it from the node_modules folder.